### PR TITLE
[Tests] Eliminate hard-coded build root from source code

### PR DESCRIPTION
**Pull request by wooksong**
This patch eliminates hard-coded build root from source code. Instead of the hard-coded 'build', this patch introduces an environment variable, NNSTREAMER_BUILD_ROOT_PATH, which is provided by the meson build script. According to this change, packaging scripts are also updated.

Signed-off-by: Wook Song <wook16.song@samsung.com>

**Self evaluation:**
1. Build test: [*]Passed [ ]Failed [ ]Skipped
2. Run test: [*]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

**Review by myungjoo — APPROVED** [review]
LGTM

**Review by anyj0527 — APPROVED** [review]
LGTM 👍🏼

File: meson.build [hunk 1]
```diff
@@ -500,6 +500,7 @@ if get_option('enable-test')
   testenv.set('NNSTREAMER_DECODERS', path_nns_plugin_decoders)
   testenv.set('NNSTREAMER_CONVERTERS', path_nns_plugin_converters)
   testenv.set('NNSTREAMER_SOURCE_ROOT_PATH', meson.source_root())
+  testenv.set('NNSTREAMER_BUILD_ROOT_PATH', meson.build_root())
```
Comment (jaeyun-jung) [inline meson.build:503]: is it necessary to update below files for testenv ?

 tests\nnstreamer_filter_extensions_common\meson.build
 tests\nnstreamer_filter_mvncsdk2\meson.build 
 tests\tizen_capi\meson.build

**Review by helloahn — APPROVED** [review]
LGTM

File: meson.build (hunk 1 above)
Comment (myungjoo) [reply meson.build:503]: Yes.. or they may rename their env variable name.

**Review by myungjoo — APPROVED** [review]

File: meson.build (hunk 1 above)
Comment (wooksong) [reply meson.build:503]: @myungjoo @jaeyun-jung Acked.

**Pull request merged**